lgc/builder/DescBuilder.cpp

```diff
@@ -280,7 +280,7 @@ Value *DescBuilder::getDescPtr(ResourceNodeType resType, unsigned descSet, unsig
     return CreateNamedCall(lgcName::SpillTable, getInt8Ty()->getPointerTo(ADDR_SPACE_CONST), {}, Attribute::ReadNone);
   };
 
-  auto GetDescriptorSetPtr = [this, node, topNode, resType, descSet]() -> Value * {
+  auto GetDescriptorSetPtr = [this, node, topNode, resType, descSet, binding]() -> Value * {
     // Get the descriptor table pointer for the set, which might be passed as a user SGPR to the shader.
```

Comment:
Change "set" in this comment.

Reply:
Done, and I fixed up other parts of the comment as well.